cmd/snap/cmd_run_test.go

```diff
@@ -76,7 +76,7 @@ func (s *SnapSuite) TestSnapRunSnapExecEnv(c *check.C) {
 	sort.Strings(env)
 	c.Check(env, check.DeepEquals, []string{
 		"PATH=${PATH}:/usr/lib/snapd",
-		"SNAP=/snap/snapname/42",
+		fmt.Sprintf("SNAP=%s/snapname/42", dirs.SnapSnapsDir),
```

Comment:
Let's please rename this variable to `dirs.SnapMountDir`. Many things there are "snap directories", and this convention matches the other cases (`info.DataDir` uses `dirs.SnapDataDir`).